example/Supfile

```diff
@@ -29,6 +29,8 @@ networks: # Groups of hosts
       - ubuntu@stg.example.com
 
   prod:
+    inventory:
+      - for i in 4 5 6; do echo "$USER@prod$i.example.com\n"; done
```

Comment:
no. I prefer `inventory` its short and sweet. I don't like any names with a "_" for our directives, if that happens, its too complicated.